services/agora/.eslintrc.cjs

```diff
@@ -104,19 +106,9 @@ module.exports = {
     ],
 
     "@typescript-eslint/no-unused-vars": ["error", { argsIgnorePattern: "^_" }],
-    //
-    // // useful to know, but too verbose https://stackoverflow.com/a/63488201/11046178
-    // "@typescript-eslint/no-floating-promises": [
-    //   "error",
-    //   {
-    //     ignoreVoid: true,
-    //   },
-    // ],
-    // "@typescript-eslint/no-misused-promises": [
-    //   "error",
-    //   {
-    //     checksVoidReturn: false,
-    //   },
-    // ],
+
+    // useful to know, but too verbose https://stackoverflow.com/a/63488201/11046178
+    "@typescript-eslint/no-floating-promises": ["error"],
```

Comment:
I was forced to do some weird changes because of this rule:

    // // useful to know, but too verbose https://stackoverflow.com/a/63488201/11046178
    // "@typescript-eslint/no-floating-promises": [
    //   "error",
    //   {
    //     ignoreVoid: true,
    //   },
    // ],
    // "@typescript-eslint/no-misused-promises": [
    //   "error",
    //   {
    //     checksVoidReturn: false,
    //   },
    // ],

But here it looks it works.

In general I'll uncomment
`    // "plugin:@typescript-eslint/recommended-type-checked",`

in a subsequent PR, and disable things that I think are useless. It's particularly useful to check switch/case completeness.